doc/source/topologies.rst

```diff
@@ -0,0 +1,137 @@
+Topologies
+==========
+
+
+Starting off
+-----------------
+Topology files are located in ``topologies`` in your streamparse project folder.
+There can be any number of topology files for your project in this directory.
+
+* topologies/my-topology.clj
+* topologies/my-other-topology.clj
+* topologies/my-third-topology.clj
+
+So on and so forth.
+
+Your topology file that describes the Directed Acyclic Graph (DAG) that will 
+be run in Storm.
```

Comment:
I'd use, "A topology file describes your topology in terms of a Directed Acyclic Graph (DAG) of Storm components, namely, Bolts and Spouts. It uses the [Clojure DSL](http://storm.apache.org/documentation/Clojure-DSL.html) for this, along with some utility Clojure functions streamparse provides."